### ISSUE TITLE
Multiple EditorJs in the DOM

### ISSUE THREAD

**Issue by hasufell** [Jungwoo-An/react-editor-js#11]
It seems I can only have one. I have virtual tabs where each of the sub-doms are hidden.

Only the first EditorJs shows.

**Comment by Jungwoo-An** [comment]
@hasufell Hi! First of all, thanks for your interest! 😍

It's difficult to understand. can you serve reproduce link?

**Comment by Jungwoo-An** [comment]
@hasufell Hi! First of all, thanks for your interest! 😍

It's difficult to understand. can you reproduce on codesandbox or others?

**Comment by hasufell** [comment]
Check out this: https://codesandbox.io/s/react-editor-js-elilp

It seems it has something to do with the hidden attribute:

```
ReactDOM.render(
  <div>
    <div hidden={true}>
      <ReactEditor />
    </div>
    <div>
      <ReactEditor />
    </div>
  </div>,
  document.getElementById("root")
);
```

**Comment by Jungwoo-An** [comment]
@hasufell Thanks!

I'll check it. sorry. (feel free to submit pr whenever you want)

**Comment by hasufell** [comment]
I thought maybe it's the `<div className="editor-js" />`, but appears it isn't: https://codesandbox.io/s/react-typescript-te1ky

It appears there must be some instance-sharing going on.

**Comment by Jungwoo-An** [comment]
@hasufell Hi!

You may need to set different `holder` property between editors. (Check [Ref](https://github.com/Jungwoo-An/react-editor-js#how-do-i-use-custom-element))

ex)

```tsx
render() {
  return (
    <div>
      <ReactEditorJs holder="holder">
        <div id="holder" />
      </ReactEditorJs>
      <ReactEditorJs holder="holder2">
        <div id="holder2" />
      </ReactEditorJs>
    </div>
  );
}
```

Thanks!

**Comment by hasufell** [comment]
IMO, this should be done automatically and the ids be randomized by default. Only `className` should be set to something like `editor-js`.

**Comment by Jungwoo-An** [comment]
I don't think so.
by default, react-editor-js use div#holder.
However, in some cases, i think can use a different tag or id.

Thanks!

**Comment by hasufell** [comment]
> I don't think so.

Why not? API-wise this is neither documented properly, nor intuitive, nor what you would expect from most react libraries.

**Comment by Jungwoo-An** [comment]
@hasufell

Thanks for your good opinion. on second thought, you are right.

I'll change `id` to random value by default. (If you don't mind, feel free to submit pull request)

**Comment by hasufell** [comment]
Sure, I'll try to write something up.

**Comment by jakekara** [comment]
Not sure if this in line with the overall code base, as I haven't looked at this project's code prior to fiddling with this, but here's one approach. 

This adds a constructor to add a holderID property that gets populated wherever the hard-coded "editor-js" had appeared.

https://github.com/jakekara/react-editor-js/commit/122a032082750f08fef12659cb5d31d6b0f1e32e

**Comment by Jungwoo-An** [comment]
resolved (#76)
